test/fixture/main.tf

```diff
@@ -1,11 +1,19 @@
+provider "azurerm" {
+  features {}
+}
+
 resource "random_id" "rg_name" {
   byte_length = 8
 }
 
+resource "azurerm_resource_group" "test" {
+  name     = "testRG-${random_id.rg_name.hex}"
+  location = var.location
```

Comment:
I may miss something but here as for "var.location", seems you deleted the "location" param in the variables.tf. So is the "var.location" still valid?

Reply:
I delete it in module's variable.tf. It is still valid in test/fixture/variable.tf

Reply:
Got. While I notice you are using "West Europe" as default values at somewhere, while for here the default value in test/fixture/variable.tf is "eastus", would you wanna make them aligned, say all use West Europe or eastus?